### PR TITLE
fix support options link

### PR DESCRIPTION
**Pull request by InsaneZein**
Fixes [RHCLOUD-29020](https://issues.redhat.com/browse/RHCLOUD-29020)

### CODE REVIEW

File: src/components/Header/Tools.tsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ const Tools = () => {
     },
     {
       title: intl.formatMessage(messages.supportOptions),
-      url: isITLessEnv ? 'https://redhatgov.servicenowservices.com/css' : 'https://access.redhat.com/support',
+      onClick: () => (window.location.href = isITLessEnv ? 'https://redhatgov.servicenowservices.com/css' : 'https://access.redhat.com/support'),
```
Comment (Hyperkid123) [inline src/components/Header/Tools.tsx:153]: The `isITLessEnv` is a boolean variable. This condition will not work.

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

File: src/components/Header/Tools.tsx (hunk 1 above)
Comment (InsaneZein) [reply src/components/Header/Tools.tsx]: is this change better?

**Review by Hyperkid123 — APPROVED** [review]

**Comment by InsaneZein** [comment]
looks like this is failing due to an unrelated cypress test: `check and uncheck favorited services.` Maybe a flaky test?

**Pull request merged**